### ISSUE TITLE
Create Tutorials "Writing Fast Task-Parallel Code Using OpenCilk"

### ISSUE THREAD

**Issue by behoppe** [OpenCilk/www.opencilk.org#223]

**Comment by AGoh2019** [comment]
Yes, please write a OpenCilk tutorial.  Using Structured Parallel Programming helps, but looking forward to full language coverage and a tool.  

It seems OpenCilk tool support is a CLI, I am requesting for a OpenCilk IDE or a extension build into the Eclipse IDE.  I just dumped VS (2026) and told MS that using VS Code doesn't meet the needs of users who require a full IDE.
